src/services/codefixes/fixStrictClassInitialization.ts

```diff
@@ -0,0 +1,141 @@
+/* @internal */
+namespace ts.codefix {
+    const fixIdAddDefiniteAssignmentAssertions = "addMissingPropertyDefiniteAssignmentAssertions";
+    const fixIdAddUndefinedType = "addMissingPropertyUndefinedType";
+    const fixIdAddInitializer = "addMissingPropertyInitializer";
+    const errorCodes = [Diagnostics.Property_0_has_no_initializer_and_is_not_definitely_assigned_in_the_constructor.code];
+    registerCodeFix({
+        errorCodes,
+        getCodeActions: (context) => {
+            const propertyDeclaration = getPropertyDeclaration(context.sourceFile, context.span.start);
+            if (!propertyDeclaration) return undefined;
+
+            const newLineCharacter = getNewLineOrDefaultFromHost(context.host, context.formatContext.options);
+            return [

```

Comment:
`getActionsForAddMissingInitializer` is the only one that can be undefined. Maybe you could use `...optionToArray(getActionsForMissingInitializer(...))` (#21628) instead of `filter`.